README.md

```diff
@@ -1,7 +1,9 @@
 ## Test262 is the official ECMAScript (ECMA-262) conformance suite.
 
+Test262 tests conformance to the continually maintained draft future ECMAScript standard found at http://tc39.github.io/ecma262/ , together with any Stage 3 or later TC39 proposals. It is maintained by Gorkem Yakin (@goyakin) with significant contributions from many in the ECMAScript community.
+
```

Comment:
In addition to a tag, I'd be OK if someone wants to, say, maintain a branch within this repository to backport certain tests to the ES6 tag, as we get new tests written that fill in the gaps for ES6. This will take a bunch of extra work to test specifically for things in ES2015 which were changed later and are errors, but if it's useful for someone, then that's fine. I don't think we need to maintain this in master, though--that would take a lot of metadata and multiple copies of tests with disagreeing results. It's not just that subsequent versions add more tests, but they also invalidate earlier test results. But for master, let's stay on the ecma262 master.